lib/kvdb/kvdb_meta.c

```diff
@@ -548,6 +557,33 @@ kvdb_meta_usage(const char *kvdb_home, uint64_t *const size)
     return err;
 }
 
+merr_t
+kvdb_meta_upgrade(struct kvdb_meta *const meta, const char *const kvdb_home)
+{
+    unsigned int omvers;
+    merr_t err;
+
+    INVARIANT(meta && kvdb_home);
```

Comment:
Probably worth splitting this up into two invariants that way it is obvious what failed.